backends/platform/sdl/switch/switch.mk

```diff
@@ -0,0 +1,19 @@
+scummvm.nro: $(EXECUTABLE)
+	mkdir -p $(srcdir)/switch_release/scummvm/data
```

Comment:
The output from these rules should be placed in the current directory, rather than in the source directory.